src/functions/quadraticIterative.jl

```diff
@@ -24,6 +24,7 @@ function (f::QuadraticIterative{R, M, V})(x::AbstractArray{R}) where {R, M, V}
 end
 
 function prox!(y::AbstractArray{R}, f::QuadraticIterative{R, M, V}, x::AbstractArray{R}, gamma::R=R(1)) where {R, M, V}
+    y .= x
```

Comment:
This is required since if `y = similar(x)` then `y` could contain `NaN`, and apparently this occurs in tests when running on Julia 1.2: it makes sense that his happens I guess, since `NaN` is a valid floating point value. 

Note that this change also has the effect of initializing `cg!` (see below) with `x` as starting point, which makes perfect sense to me.